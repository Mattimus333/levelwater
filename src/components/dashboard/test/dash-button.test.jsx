import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DashButton from '../dash-button.jsx';

describe('dash button', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<DashButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
