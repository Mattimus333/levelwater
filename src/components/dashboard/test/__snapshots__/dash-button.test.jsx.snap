exports[`dash button renders a snapshot 1`] = `
<div
  id="dash-button">
  <button
    className="ui button" />
</div>
`;
